test/unit/specs/components/transactions/LiTransaction.spec.js

```diff
@@ -25,14 +25,20 @@ describe(`LiTransaction`, () => {
   })
 
   it(`Should print the hour only if the same day`, () => {
-    expect(LiTransaction.computed.date({ time: Date.now() })).toEqual(
-      `00:00:42`
+    expect(
+      LiTransaction.computed.date(
+        { time: new Date(420000).toISOString() }
+      )
+    ).toEqual(
+      `00:07:00`
     )
   })
 
   it(`Should print the datetime if we are in a different day`, () => {
     expect(
-      LiTransaction.computed.date({ time: Date.now() + 999999999 })
-    ).toEqual(`Jan 12th 1970 13:47:21`)
+      LiTransaction.computed.date(
+        { time: new Date(4200000 + 999999999).toISOString() }
```

Comment:
I still think is more readable with new Date(Date.now() + 999..) it's basically for the reader to know the rendering changes when you change day
Even better would be to change the 9999....99 with a meaningful multiplication.

that's just becasuse is hard to notice that 
42000 < 420000 < 4200000

Reply:
the `day * 2` is in the lcdClientMock if you want to recycle from there (before it's dropped)

Reply:
updated